.github/workflows/build.yml

```diff
@@ -188,8 +227,72 @@ jobs:
-            docker run --rm --env-file api.env ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:api npm run db:drop
-            docker run --rm --env-file api.env -v ./platform-explorer/packages/indexer/migrations:/migrations owl352/refinery_cli_docker refinery migrate -e DATABASE_URL -p /migrations
             docker run -d -p 3005:3005 --restart always --env-file api.env --name platform-explorer-api ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:api
             sleep 3
             docker run -d --env-file api.env --restart always --name platform-explorer-indexer ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:indexer ./indexer
+            
+  create_tag:
+    runs-on: ubuntu-latest
+    needs: [build_api, build_indexer, test_data_contract, test_api, test_frontend]
+    
+    if: github.event_name != 'pull_request' || github.event.pull_request.merged == true
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: 'Get Previous tag'
+        id: previoustag
+        uses: "WyriHaximus/github-action-get-previous-tag@v1"
+        with:
+          fallback: 1.0.0
+
+      - name: get-npm-version
+        id: package-version
+        uses: martinbeentjes/npm-get-version-action@v1.3.1
+        with:
+          path: ./
+
+      - name: 'Generate changelog'
+        id: changelog
+        uses: heinrichreimer/action-github-changelog-generator@v2.3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }} 
+          headerLabel: "# 📑 Changelog"
+          breakingLabel: '### 💥 Breaking'
+          enhancementLabel: '### 🚀 Enhancements'
+          bugsLabel: '### 🐛 Bug fixes'
+          deprecatedLabel: '### ⚠️ Deprecations'
+          removedLabel: '### 🔥 Removals'
+          securityLabel: '### 🛡️ Security'
+          issuesLabel: '### 📁 Other issues'
+          prLabel: '### 📁 Other pull requests'
+          addSections: '{"indexer":{"prefix":"### 🧩 Indexer","labels":["indexer"]},"api":{"prefix":"###📊 API","labels":["backend"]},"frontend":{"prefix":"###👨‍💻 frontend","labels":["frontend"]}}'
+          issues: true
+          issuesWoLabels: true
+          pullRequests: true
+          prWoLabels: true
+          author: true
+          unreleased: true
+          compareLink: true
+          stripGeneratorNotice: true
+          verbose: true
+
+      - name: "🖨️ Print changelog to console"
+        run: cat CHANGELOG.md
+
+      - name: "📤 Upload changelog"
+        uses: actions/upload-artifact@v4
+        with:
+          name: "Changelog"
+          path: CHANGELOG.md
+
+      - name: Create a GitHub release
+        if: ${{steps.package-version.outputs.current-version != steps.previoustag.outputs.tag }}
+        uses: ncipollo/release-action@v1
+        with:
+          tag: ${{ steps.package-version.outputs.current-version}}
+          name: Release ${{ steps.package-version.outputs.current-version}}
+          body: ${{steps.changelog.outputs.changelog}}
```

Comment:
It should only show info about latest release (now all is shown)

Reply:
✅